netdisco/discoverables/heos.py

```diff
@@ -0,0 +1,10 @@
+"""Discover Heos devices."""
+from . import SSDPDiscoverable
+
+
+class Discoverable(SSDPDiscoverable):
+    """Add support for discovering DLNA services."""
+
+    def get_entries(self):
+        """Get all the HEOS devices."""
+        return self.find_by_st("urn:schemas-denon-com:device:ACT-Denon:1")
```

Comment:
This won't identify receivers or other denon devices?

Reply:
According to the [CLI Specification](http://rn.dmglobal.com/euheos/HEOS_CLI_ProtocolSpecification.pdf) (page 6), this is the search target for HEOS-enabled devices.  It will include Denon and Marantz receivers, amps, and speakers that have HEOS, which is intentional.  It should not match older devices that aren't HEOS enabled.

Reply:
That could possibly be clarified.